### DISCUSSION TITLE
macOS 3.3.2 problems with using "highlight" and "text color" with essay-question

### DISCUSSION THREAD

**Discussion by moehzhaw** [SafeExamBrowser/seb-mac#407] [Q&A]
Hi
We have the following setting and problems.
**= Does anybody know this problem?**

***
LMS: Moodle 4.2.7
SEB-Version: macOS 3.3.2
on: MacBook Air, M1, 2020, macOS Sonoma 14.4.1 and MacBook Pro, 2020, macOS Sonoma 14.5

**relates to: Question type: essay**

***

**problem 1**
- behaviour: When using the highlighting there is a time delay = see 2024-05-22_SEB_Issue_Textformat_Delay_macOS
- expected: no time delay = immediate highlighing

**problem 2:**
- behaviour: When using the text format (text color) the prompt jumps to the end of the textfield = see 2024-05-22_SEB_Issue_Textformat_Prompt-Jump_macOS
- expected: no jump of the prompt = prompt stays 

👉Link to [OneDrive with videos](https://zhaw-my.sharepoint.com/:f:/g/personal/moeh_zhaw_ch/Eo8vNDpAn-ZKnQ3HcF3ahicB07WRbTOAxrHmQaTZ-gU0qQ?e=8Lhlf1) = videos are available till 27th of June 2024

**Comment by danschlet** [comment]
Can you please check if this also happens in Safari on the same Macs with the same quizzes?

I don't have that Atto plugins, cannot reproduce it right now.

**Reply by moehzhaw** [reply]
Yes, it is the same with MacBook Pro, 2020, macOS Sonoma 14.5 in Safari 17.5 (19618.2.12.11.6) .

**Reply by moehzhaw** [reply]
And yes, we do use Atto... and ETH's Moodle operates with Tiny (=correct?)
--> Could this be the reason for these problems?

**Reply by moehzhaw** [reply]
Using SEB-Version 3.5.4 for macOS and Tiny-Editor --> There is no problem as mentioned anymore...
